pages/notification.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import personData from '../mock/person.json';
+import NavBar from '@components/NavBar';
+
+export default function notification() {
+  return (
+    <div className="container">
+      <NavBar personData={personData} />
+      <div className="notification">
+        <ul>
+          <li>
+            <div className="notificationArea">
+              <h4>Rohit has requested RDS 200 from you</h4>
+              <div className="buttons">
+                <button className="approve">Approve</button>
+                <button className="discard">Discard</button>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="notificationArea">
+              <h4>Rohit has requested RDS 200 from you</h4>
+              <div className="buttons">
+                <button className="approve">Approve</button>
+                <button className="discard">Discard</button>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="notificationArea">
+              <h4>Rohit has requested RDS 200 from you</h4>
+              <div className="buttons">
+                <button className="approve">Approve</button>
+                <button className="discard">Discard</button>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="notificationArea">
+              <h4>Rohit has requested RDS 200 from you</h4>
+              <div className="buttons">
+                <button className="approve">Approve</button>
+                <button className="discard">Discard</button>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="notificationArea">
+              <h4>Rohit has requested RDS 200 from you</h4>
+              <div className="buttons">
+                <button className="approve">Approve</button>
+                <button className="discard">Discard</button>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="notificationArea">
+              <h4>Rohit has requested RDS 200 from you</h4>
+              <div className="buttons">
+                <button className="approve">Approve</button>
+                <button className="discard">Discard</button>
+              </div>
+            </div>
+          </li>
```

Comment:
Can we please refactor this? `li` has a similar pattern throughout. Maybe create a new function that returns this `li` and call that function 4-5 times, or create a component and call that component 4-5 times.

That will make this code much more clean and readable.

Reply:
Ok thnks. I will make the changes.

Reply:
@sumitd94 Changes has been made. Please check.